openfe/tests/setup/test_network_planning.py

```diff
@@ -244,6 +244,83 @@ def scorer(mapping):
             scorer=scorer
         )
 
+@pytest.fixture(scope='session')
+def generate_minimal_redundant_network(toluene_vs_others):
```

Comment:
for the fixture, maybe give it a noun name instead of a verb? e.g, `minimal_redundant_network`

Reply:
done

Reply:
I'm not sure I can do this because I'll need to pull from `minimal_spanning_network` which is under the `session` scope? I left it all under the `session` scope for now.

Reply:
poor copy-pasting on my part, again..